.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: php
+
+sudo: false
+
+cache:
+  directories:
+    - "$HOME/.composer/cache"
```

Comment:
it does if you test against stable version (for dev versions, it does not indeed because the cache will miss all the time).

The subtle issues in symfony were only because of the weird logic applied to create subtree packages locally.